app/src/pages/playground/PlaygroundChatTemplate.tsx

```diff
@@ -243,7 +243,9 @@ function SortableMessageItem({
         }
         extra={
           <Flex direction="row" gap="size-100">
-            <CopyToClipboardButton text={message.content} />
+            {message.content !== undefined && (
```

Comment:
same here, for undefined / null would typically do equality with != null unless you have good reason not to